packages/api-client/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@beyondessential/tupaia-api-client",
+  "version": "1.0.15",
+  "description": "API client for connecting to Tupaia APIs",
+  "main": "dist/index.js",
+  "types": "./dist/**/**.d.ts",
+  "license": "SEE LICENSE IN LICENSE",
+  "scripts": {
+    "build": "babel src --out-dir dist --ignore \"src/__tests__/**\" --extensions .ts --config-file \"../../.babelrc-ts.js\"",
+    "lint": "tsc; eslint . --ext .ts",
+    "lint:fix": "yarn lint --fix",
+    "build:typedefs": "tsc",
+    "test": "jest --runInBand",
+    "test:coverage": "yarn test --coverage",
+    "publish:run": "yarn publish --no-git-tag-version",
```

Comment:
`yarn publish` is a very handly tool that increments the version, commits the change and tags the commit. However, it doesn't know we are using a monorepo, and we don't want tags for this package to be pushed into the monorepo. So I wrote publish:run to be used instead.